bower.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "n-email-article",
   "dependencies": {
-    "n-ui-foundations": "^1.0.1",
-    "n-notification": "^5.0.5",
-    "o-forms": "^2.0.5"
+    "n-ui-foundations": "^2.0.0-beta",
+    "n-notification": "^6.0.0-beta",
```

Comment:
for the sake of cleanliness, both of these have been released so we can get rid of the 'beta' :D